mindmeld/query_factory.py

```diff
@@ -15,10 +15,8 @@
 
 from __future__ import absolute_import, unicode_literals
 
-import nltk
-
 from . import ser as sys_ent_rec
-
+from .stemmers import EnglishNLTKStemmer
```

Comment:
Import order: `.stemmers` should be after `.core`

Reply:
resolved